tests/EvmAcceptanceTests/helper/Web3Helper.js

```diff
@@ -29,6 +29,15 @@ class Web3Helper {
 
         return deployedContract;
     }
+
+    async getCommonOptions(base = {}) {
```

Comment:
One note. I found it more idiomatic in javascript to have a variable and extend it with properties and methods instead of writing a class for it. something like `GeneralHelper.js` what do you think? It makes more sense because most of the functions are static functions and they don't have any related state. If you agree, please revert web3js helper to its previous state.

Reply:
Sure, will do. What about making these methods static instead?

Reply:
Sounds good. I guess all of them become static including the variables. making them static, or what I suggested, eliminate the need of instantation of Web3Helper class.